### PR TITLE
Used klog for katib - part 1.

### PR DESCRIPTION
**Pull request by gyliu513**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes part of https://github.com/kubeflow/katib/issues/506

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note
Used klog for katib.
```

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/katib/526)

### CODE REVIEW

**Comment by gyliu513** [comment]
I will follow up with another PR to update files which using `ZapLogger`.

**Comment by johnugeorge** [comment]
@gyliu513 Can you reply to this https://github.com/kubeflow/katib/issues/506#issuecomment-491762379 ?

**Comment by gaocegege** [comment]
This PR mainly replace log in cmd with klog. I am not sure if we should replace zaplogger in controller. Persoanlly LGTM

Please rebase

**Comment by gyliu513** [comment]
@gaocegege Please see comments here https://github.com/kubeflow/katib/pull/526#issuecomment-492943766

**Comment by gaocegege** [comment]
Yeah I know you will do it in another PR. But I am not sure if we need it. You need to persuade Johnu :smile:

**Comment by gaocegege** [comment]
But personally, I think we could use klog.

File: pkg/api/v1alpha2/api.pb.gw.go [hunk 1]
```diff
@@ -18,8 +18,8 @@ import (
 	"golang.org/x/net/context"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/codes"
-	"google.golang.org/grpc/grpclog"
 	"google.golang.org/grpc/status"
+	"k8s.io/klog"
```
Comment (hougangliu) [inline pkg/api/v1alpha2/api.pb.gw.go]: api.pb.gw.go files are generated based on api.proto, I don't think we should modify the files manually

File: pkg/api/v1alpha1/api.pb.gw.go [hunk 2]
```diff
@@ -492,14 +492,14 @@ func RegisterManagerHandlerFromEndpoint(ctx context.Context, mux *runtime.ServeM
...
-				grpclog.Printf("Failed to close conn to %s: %v", endpoint, cerr)
+				klog.Fatalf("Failed to close conn to %s: %v", endpoint, cerr)
 			}
 			return
 		}
 		go func() {
 			<-ctx.Done()
 			if cerr := conn.Close(); cerr != nil {
-				grpclog.Printf("Failed to close conn to %s: %v", endpoint, cerr)
+				klog.Fatalf("Failed to close conn to %s: %v", endpoint, cerr)
```
Comment (hougangliu) [inline pkg/api/v1alpha1/api.pb.gw.go]: refer to below comment

File: pkg/api/v1alpha2/api.pb.gw.go (hunk 1 above)
Comment (gyliu513) [reply pkg/api/v1alpha2/api.pb.gw.go]: DOne

**Comment by hougangliu** [comment]
@gyliu513 thanks!
/lgtm
@johnugeorge WDYT?

**Comment by johnugeorge** [comment]
Please rebase the changes 
/lgtm

**Pull request merged**